lexical-parse-float/src/slow.rs

```diff
@@ -572,7 +572,7 @@ pub fn byte_comp<F: RawFloat, const FORMAT: u128>(
 
     // Now, create a scaling factor for the digit count.
     let mut factor = Bigfloat::from_u32(1);
-    factor.pow(format.radix(), sci_exp.abs() as u32).unwrap();
```

Comment:
I see you made this change to solve linter errors. Is the linter failing upstream as well? I just want to make sure we aren't making unexpected changes to the upstream project.

Reply:
It is not failing upstream. However, I believe the linter they're using is from a public library and it has been updated since they last ran it. Without making any changes to their code, my github action was failing previously because the linter suggested I change the abs() function to unsigned abs() on both of these lines.